### ISSUE TITLE
pick-list and value-list do not pass latest aXe accessibility tests

### ISSUE THREAD

**Issue by driskull** [Esri/calcite-components#5848]
### Summary

pick-list and value-list do not pass latest aXe accessibility testing when aXe was bumped from 4.4.3 to 4.5.2

https://github.com/Esri/calcite-components/pull/5841

### Actual Behavior

Here is a log of the failing tests.

It seems that the parent component has a role of "menu" on the host whereas the role for the "menuitemcheckbox" is internal to the child item.

The other potential issue is that the child might need to be a direct child of the parent.

```
Summary of all failing tests
 FAIL  src/components/value-list/value-list.e2e.ts (30.01 s)
  ● calcite-value-list › is accessible

    expect(received).toHaveNoViolations(expected)

    Expected the HTML found at $('calcite-value-list') to have no violations:

    <calcite-value-list aria-busy="false" aria-owns="calcite-value-list-item-a08936b2-cc3c-6ae5-57b7-739ee9c6897b" role="menu" calcite-hydrated="">

    Received:

    "Certain ARIA roles must contain particular children (aria-required-children)"

    Fix any of the following:
      Element has children which are not allowed (see related nodes)
      Element has no aria-busy="true" attribute

    You can find more information on this issue here: 
    https://dequeuniversity.com/rules/axe/4.5/aria-required-children?application=axeAPI

      66 |       getTag(componentTagOrHTML)
      67 |     )
    > 68 |   ).toHaveNoViolations();
         |     ^
      69 | }
      70 |
      71 | /**

      at accessible (src/tests/commonTests.ts:68:5)

 FAIL  src/components/pick-list/pick-list.e2e.ts (22.82 s)
  ● calcite-pick-list › is accessible

    expect(received).toHaveNoViolations(expected)

    Expected the HTML found at $('calcite-pick-list') to have no violations:

    <calcite-pick-list aria-busy="false" aria-owns="" role="menu" calcite-hydrated="">
      <calcite-pick-list-item label="Sample" value="one" disable-deselect="" icon="circle" intl-remove="Remove" calcite-hydrated=""></calcite-pick-list-item>
    </calcite-pick-list>

    Received:

    "Certain ARIA roles must contain particular children (aria-required-children)"

    Fix any of the following:
      Element has children which are not allowed (see related nodes)
      Element has no aria-busy="true" attribute

    You can find more information on this issue here: 
    https://dequeuniversity.com/rules/axe/4.5/aria-required-children?application=axeAPI

      66 |       getTag(componentTagOrHTML)
      67 |     )
    > 68 |   ).toHaveNoViolations();
         |     ^
      69 | }
      70 |
      71 | /**

      at accessible (src/tests/commonTests.ts:68:5)

 FAIL  src/components/pick-list-item/pick-list-item.e2e.ts (6.949 s)
  ● calcite-pick-list-item › is accessible

    expect(received).toHaveNoViolations(expected)

    Expected the HTML found at $('calcite-pick-list') to have no violations:

    <calcite-pick-list aria-busy="false" aria-owns="" role="menu" calcite-hydrated="">
      <calcite-pick-list-item label="test" description="a number" value="one" disable-deselect="" icon="circle" intl-remove="Remove" calcite-hydrated=""></calcite-pick-list-item>
    </calcite-pick-list>

    Received:

    "Certain ARIA roles must contain particular children (aria-required-children)"

    Fix any of the following:
      Element has children which are not allowed (see related nodes)
      Element has no aria-busy="true" attribute

    You can find more information on this issue here: 
    https://dequeuniversity.com/rules/axe/4.5/aria-required-children?application=axeAPI

      66 |       getTag(componentTagOrHTML)
      67 |     )
    > 68 |   ).toHaveNoViolations();
         |     ^
      69 | }
      70 |
      71 | /**

      at accessible (src/tests/commonTests.ts:68:5)
          at runMicrotasks (<anonymous>)
      at Object.<anonymous> (src/components/pick-list-item/pick-list-item.e2e.ts:12:5)

 FAIL  src/components/value-list-item/value-list-item.e2e.ts (6.123 s)
  ● calcite-value-list-item › is accessible

    expect(received).toHaveNoViolations(expected)

    Expected the HTML found at $('calcite-value-list') to have no violations:

    <calcite-value-list aria-busy="false" aria-owns="calcite-value-list-item-84578da1-834c-dd25-8631-d90c798322c1" role="menu" calcite-hydrated="">

    Received:

    "Certain ARIA roles must contain particular children (aria-required-children)"

    Fix any of the following:
      Element has children which are not allowed (see related nodes)
      Element has no aria-busy="true" attribute

    You can find more information on this issue here: 
    https://dequeuniversity.com/rules/axe/4.5/aria-required-children?application=axeAPI

      66 |       getTag(componentTagOrHTML)
      67 |     )
    > 68 |   ).toHaveNoViolations();
         |     ^
      69 | }
      70 |
      71 | /**

      at accessible (src/tests/commonTests.ts:68:5)
          at runMicrotasks (<anonymous>)
      at Object.<anonymous> (src/components/value-list-item/value-list-item.e2e.ts:12:5)

 FAIL  src/components/pick-list-group/pick-list-group.e2e.ts (5.173 s)
  ● calcite-pick-list-group › is accessible

    expect(received).toHaveNoViolations(expected)

    Expected the HTML found at $('calcite-pick-list') to have no violations:

    <calcite-pick-list aria-busy="false" aria-owns="" role="menu" calcite-hydrated="">

    Received:

    "Certain ARIA roles must contain particular children (aria-required-children)"

    Fix any of the following:
      Element has children which are not allowed (see related nodes)
      Element has no aria-busy="true" attribute

    You can find more information on this issue here: 
    https://dequeuniversity.com/rules/axe/4.5/aria-required-children?application=axeAPI

      66 |       getTag(componentTagOrHTML)
      67 |     )
    > 68 |   ).toHaveNoViolations();
         |     ^
      69 | }
      70 |
      71 | /**

      at accessible (src/tests/commonTests.ts:68:5)
          at runMicrotasks (<anonymous>)
      at Object.<anonymous> (src/components/pick-list-group/pick-list-group.e2e.ts:14:5)


Test Suites: 5 failed, 1 skipped, 105 passed, 110 of 111 total
Tests:       5 failed, 21 skipped, 2463 passed, 2489 total
```

### Expected Behavior

aXe tests should pass for these two components.

The tests should pass when no longer skipped.

### Reproduction Sample

https://developers.arcgis.com/calcite-design-system/components/value-list/

### Reproduction Steps

1. Remove the skip of the tests that were skipped in the linked PR.
2. Notice that they do not pass.

### Reproduction Version

1.0.0-next.647

### Working W3C Example/Tutorial

https://www.w3.org/WAI/ARIA/apg/patterns/menu/

### Relevant Info

https://github.com/Esri/calcite-components/pull/5841

https://dequeuniversity.com/rules/axe/4.5/aria-required-children?application=axeAPI

### Esri team

Calcite (dev)

**Comment by brittneytewks** [comment]
Removed "needs triage" since a priority is set and estimate is indicated

**Comment by geospatialem** [comment]
With the `list` accessibility refactor, and deprecated `value-list` and `pick-list` components - will focus attention on `list`, in particular as we work towards parity with the deprecated components in #6430. For now, closing out this issue as we might need to do some larger refactors on the deprecated components, where we could instead focus efforts on the `list` component parity.
